src/main/java/net/snowflake/client/config/SFClientConfigParser.java

```diff
@@ -86,14 +86,16 @@ public static String getConfigFilePathFromJDBCJarLocation() {
       String jarPath =
           SnowflakeDriver.class.getProtectionDomain().getCodeSource().getLocation().getPath();
 
-      // remove /snowflake-jdbc-3.13.29.jar from the path.
-      String updatedPath = jarPath.substring(0, jarPath.lastIndexOf("/"));
+      // remove /snowflake-jdbc-3.13.29.jar and anything that follows it from the path.
+      String updatedPath = new File(jarPath).getParentFile().getPath();
```

Comment:
Thanks for the fix. 

Does it make sense to have an exception handler for this method? In case we run into any exceptions - can we ignore to load the config from the Snowflake jar location?

Reply:
yeah I think that makes sense. I added a try/catch to return an empty path if there's an exception so it will move on to the next loading step.